engine/runtime/src/main/java/org/enso/interpreter/runtime/error/WithWarnings.java

```diff
@@ -21,6 +21,9 @@
 @ExportLibrary(WarningsLibrary.class)
 @ExportLibrary(ReflectionLibrary.class)
 public final class WithWarnings implements TruffleObject {
+
+  private static final int MAX_WARNINGS_LIMIT = 100;
```

Comment:
Obviously this should be a compilation constant, if we want to use it in PE mode. Should it become an option, then you have to place the value in `EnsoContext`.

Reply:
Doesn't `static final int` field imply `@CompilationFinal`?

Reply:
`static final int` is indeed treated as compilation constant by Truffle PE.

Reply:
It's `private final int` now, which I guess covers it?